KProcessHacker/main.c

```diff
@@ -395,30 +395,33 @@ NTSTATUS KpiPopulateKnownDllExtents(
     mappedBase = NULL;
     mappedSize = 0;
 
-    InitializeObjectAttributes(&objectAttributes,
-                               SectionName,
-                               OBJ_KERNEL_HANDLE,
-                               NULL,
-                               NULL);
-
-    status = ZwOpenSection(&sectionHandle,
-                           SECTION_MAP_READ | SECTION_QUERY,
-                           &objectAttributes);
+    InitializeObjectAttributes(
+        &objectAttributes,
+        SectionName,
+        OBJ_KERNEL_HANDLE,
+        NULL,
+        NULL
+        );
+
+    status = ZwOpenSection(
+        &sectionHandle,
+        SECTION_MAP_READ | SECTION_QUERY,
+        &objectAttributes
+        );
+
     if (!NT_SUCCESS(status))
-    {
-        sectionHandle = NULL;
```

Comment:
Please don't trust that garbage won't make it into the output parameter a failed call, while normally this isn't a big deal and in most circumstances the Windows API won't populate it with garbage on a failed call, it's good practice for kernel code to clear the output data that might land in cleanup later. The reasoning behind this pattern is to help ensure robustness of the kernel code.